lang/py/avro/schema.py

```diff
@@ -836,6 +912,11 @@ def to_json(self, names=None):
             to_dump.append(schema.to_json(names))
         return to_dump
 
+    def to_canonical_json(self, names):
```

Comment:
…

Reply:
Yes. Addressed in eacfdd6a0cd6de9b4e15701d08d9e994f09be310